spec/workflow/reference_path_spec.rb

```diff
@@ -50,19 +50,27 @@
 
   describe "#set" do
     let(:payload) { "$" }
-    let(:input) { {} }
+    let(:input) { { "old" => "key"} }
```

Comment:
Rubocop has a good point
```suggestion
    let(:input) { {"old" => "key"} }
```